android/src/main/java/com/amplitude/skylab/DefaultSkylabClient.java

```diff
@@ -120,23 +116,6 @@ public void start(SkylabUser skylabUser, long timeoutMs) {
         }
     }
 
-    public String loadFromStorage() {
```

Comment:
we don't need local storage anymore? Or we already have a alternative for this?

Reply:
The `SharedPrefsStorage` handles the API for using `SharedPreferences`. When `SharedPrefsStorage` is initialized with the Android `Context`, it loads the shared preferences instance separately.

The actual use of this function was loading the enrollment ID only, which is no longer necessary since we don't use it.